api/twilioapi/twilioapi.py

```diff
@@ -137,7 +139,7 @@ async def get_partial_result(
 async def stream_ai_response(ai_response):
     UnstableSpeechResultList = []
     for word in ai_response.split():
-        await sleep(0.2)
+        await sleep(0.5)
```

Comment:
is this how long it waits before generating response? 
0.5 seconds? What is the lowest that we can set for testing?

Reply:
it's delay between transcribed speech sent via sockets for live speech transcript. 0.5 is the best value